auctions/english-auction/src/events.sw

```diff
@@ -0,0 +1,28 @@
+library events;
+
+dep data_structures;
+
+use data_structures::{Asset, Auction};
+use std::identity::Identity;
+
+pub struct AuctionCancelEvent {
+    auction: Auction,
```

Comment:
You probably don't need to emit the auction itself since the front end can query by the id

Reply:
I agree